terraform/ndelius-test-automation.tf

```diff
@@ -1,4 +1,4 @@
-module "ndelius-test-automation" {
+module "ndelius-test-automation-swestb" {
```

Comment:
Please can that back to `ndelius-test-automation`. Module needs to be the same as the name of the repo. This will break existing access.